web/skins/classic/views/event.php

```diff
@@ -154,7 +154,11 @@
 ?>
         <a class="btn btn-normal" href="<?php echo $Event->getStreamSrc(array('mode'=>'mp4'),'&amp;')?>" download><i class="fa fa-download"></i></a>
 <?php
-  } 
+  } else {
+?>
+        <button id="videoEvent" class="btn btn-normal" data-toggle="tooltip" data-toggle="tooltip" data-placement="top" title="<?php echo translate('Download') ?>" onclick="videoEvent();"><i class="fa fa-download"></i></button>
```

Comment:
inline JS onclick events are deprecated. Instead you should bind the button id to a function, like the other buttons on this page, here: https://github.com/ZoneMinder/zoneminder/blob/master/web/skins/classic/views/js/event.js#L937